one-app-statics/postinstall.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+// eslint-disable-next-line no-console
+console.log('🚨 🚨 🚨 one-app-statics is not intended to be installed using npm, please refer to the README for more details!🚨 🚨 🚨 ');
```

Comment:
Not sure if we need to have this. It would limit the package strictly for CDN usage - I get the entry point throwing however to prevent the package from installing might be restrictive for future endeavors with `one-app` local/static usage.

Reply:
This specific package `@americanexpress/one-app-statics` is strictly for CDN usage, it doesn't serve any other purpose. It is a package inside one-app not one-app itself if that makes sense?

Reply:
If that's the case, the PR looks good.